test/performance/logs/parameters.yml

```diff
@@ -8,7 +8,7 @@ number_monitored_logs: 1000
 # Number of metrics to be sent or number of log lines being written  each minute
 values_per_minute: "<values_per_minute>"
 # Number of seconds the agent should run and collect the metrics. In this case, 10 minutes
```

Comment:
nit: update comment

Reply:
Thanks. I have no reason or recollection why I changed it back to 60 instead of 600.